Makefile

```diff
@@ -9,11 +9,13 @@ VERSION=`head -1 ChangeLog.txt | sed 's/^SASyLF version \(.*\).$$/\1/'`.v`date +
 
 default: test
 
+build : SHELL:=/bin/bash
 build :
 	(cd src && cd edu && cd cmu && cd cs && cd sasylf && cd parser; javacc parser.jj)
 	mkdir -p bin
-	(cd src && javac -source 1.8 -target 1.8 -classpath ../bin:. -d ../bin edu/cmu/cs/sasylf/Main.java edu/cmu/cs/sasylf/term/UnitTests.java)
-	jar cmf sasylf.mf SASyLF.jar ChangeLog.txt -C bin edu -C library org
+	(cd src && javac -cp ../lib/*:../bin:. -source 1.8 -target 1.8 -d ../bin edu/cmu/cs/sasylf/Main.java edu/cmu/cs/sasylf/term/UnitTests.java)
```

Comment:
I don't think "\*" in a -cp argument does what you want here.  Globbing works at the shell level; unless bash is doing something strange, "../lib/\*:../bin:." should be looking for a file ../lib/A/bin/:." where A ends with colon and two dots